### ISSUE TITLE
Why implement the cross entropy loss by our own in multi-box loss other than the loss in pytorch?

### ISSUE THREAD

**Issue by USTClj** [amdegroot/ssd.pytorch#230]
Hi every. Why do we re-implement the a loss function to work for the hard negative sample mining? Why not use the `F.cross_entropy` twice. Is it related to the forward and backward in pytorch?
